src/test/java/be/seeseemelk/mockbukkit/entity/EntityMockTest.java

```diff
@@ -10,14 +10,19 @@
 import java.util.Optional;
 import java.util.Set;
 import java.util.UUID;
+import java.util.concurrent.atomic.AtomicBoolean;
 import java.util.stream.Collectors;
 
+import be.seeseemelk.mockbukkit.*;
```

Comment:
Not a big fan of wildcard imports.

Reply:
Sorry, it was automatically generated by IntelliJ (as there were more than 3 import I think)